R/path.R

```diff
@@ -0,0 +1,16 @@
+#' Show Exchange Path of TV
+#'
+#' The main R session uses this path to exchange information with the TV session.
+#' Exported for debugging.
```

Comment:
For debugging we can use `tv:::path()` ?

Maybe it's sufficient to document with `@keywords internal` ?

Reply:
true, changed it.